apps/common/models/i-casework-getcase.js

```diff
@@ -29,9 +28,16 @@ module.exports = class DocumentModel extends Model {
   }
 
   fetch() {
-    const options = this.requestConfig({});
-    options.qs = this.prepare();
-    options.method = 'GET';
-    return this.request(options);
+    const params = {
+      url: this.url(),
+      method: 'GET',
+      params: this.prepare()
+    };
+    return this._request(params).then(response => {
+      return this.parse(response.data);
+    })
+      .catch(err => {
+        return err;
+      });
   }
 };
```

Comment:
Nice work

out of scope but again this could be improved with async await.  Leave this with me